aws/session/shared_config_test.go

```diff
@@ -507,6 +528,15 @@ func TestLoadSharedConfigFromFile(t *testing.T) {
 				S3UseARNRegion: true,
 			},
 		},
+		{
+			Profile: "sso-session-success",
```

Comment:
whats the difference between this test and the other test above that tests `sso-session-success`?